lms/templates/courseware_search/search_list.underscore

```diff
@@ -10,11 +10,11 @@
     <% if (hasMoreResults) { %>
         <a class="search-load-next" href="javascript:void(0);">
             <%= interpolate(
-                ngettext("Load next %(num_items) result", "Load next %(num_items) results", pageSize),
+                ngettext("Load next %(num_items) result", "Load next %(num_items)s results", pageSize),
```

Comment:
By adding the 's' character, `%(NUM_ITEMS)` you saw in the previous screenshot is replaced by a concrete number, as you can see in the following screenshot. `(num_items)` is just a name that's provided here, there should be `%s` in here, nevertheless.
![screen shot 2015-04-07 at 10 44 14](https://cloud.githubusercontent.com/assets/6833568/7019709/311123d2-dd13-11e4-8793-1739fb6e603e.png)